File: linguagem_c/ex03.c
#include <stdio.h>

int main() {
    int quant_jogos;
    const float preco_jogo = 19.90;
    const float salario_base_perc = 0.5;
    const float bonus_percentual = 0.08;
    const int quant_jogos_bonus = 15;

    printf("Digite a quantidade de jogos vendidos esse mês por Catarina: ");
    scanf("%d", &quant_jogos);

    float valor_total_vendas = quant_jogos * preco_jogo;

    float valor_bonus = 0.0;
    if (quant_jogos >= quant_jogos_bonus) {
        valor_bonus = valor_total_vendas * bonus_percentual;
    }

    float salario_base = valor_total_vendas * salario_base_perc;
    float salario_total = salario_base + valor_bonus;

    printf("%.2f\n", valor_total_vendas);
    printf("%.2f\n", valor_bonus);
    printf("%.2f\n", salario_total);

    return 0;
}

File: linguagem_java/ex03.java
import java.util.Scanner;

public class ex03 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int quantidadeJogos;
        final float precoJogo = 19.90f;
        final float salarioBasePercen = 0.5f;
        final float bonusPercen = 0.08f;
        final int quantidadeJogosParaBonus = 15;

        System.out.print("Quantidade de jogos vendidos por Catarina nesse mês: ");
        quantidadeJogos = scanner.nextInt();

        float valorTotalVendas = quantidadeJogos * precoJogo;

        float valorBonus = 0.0f;
        if (quantidadeJogos >= quantidadeJogosParaBonus) {
            valorBonus = valorTotalVendas * bonusPercen;
        }

        float salarioBase = valorTotalVendas * salarioBasePercen;
        float salarioTotal = salarioBase + valorBonus;

        System.out.printf("%.2f\n", valorTotalVendas);
        System.out.printf("%.2f\n", valorBonus);
        System.out.printf("%.2f\n", salarioTotal);

        scanner.close();
    }
}
